src/store/Configuration/actions.ts

```diff
@@ -2,32 +2,14 @@ import axios from '@/axios-auth'
 
 export default {
   /** Fetches the configuration from the web server and, if successful, triggers some mutations. */
-  loadConfiguration (context, applicationUrl: string): Promise<any> {
-    // fetch config from server
-    // eg, http://localhost:8080/business/config/configuration.json
-    // eg, https://dev.bcregistry.ca/business/config/configuration.json
-    const url = `${applicationUrl}config/configuration.json`
-    const headers = {
-      'Accept': 'application/json',
-      'ResponseType': 'application/json',
-      'Cache-Control': 'no-store'
-    }
+  loadConfiguration (context): Promise<any> {
     // need to return a promise because action is called via dispatch
-    return new Promise((resolve, reject) => {
-      axios.get(url, { headers })
-        .then((response) => {
-          if (!response.data) {
-            reject(new Error('Invalid configuration.json'))
-          } else {
-            context.commit('setConfiguration', response.data)
-            context.commit('setSessionVariables', response.data)
-            context.commit('setAxiosBaseUrl', context.getters.getLegalApiUrl)
-            resolve(response.data)
-          }
-        })
-        .catch(error => {
-          reject(new Error('Error: ' + error))
-        })
+    return new Promise((resolve) => {
+      context.commit('setConfiguration', process.env)
+      context.commit('setSessionVariables', process.env)
+      context.commit('setAxiosBaseUrl', context.getters.getLegalApiUrl)
+
+      resolve(process.env)
```

Comment:
I'll let @jonathan-longe speak to whether Entities Team needs a follow-up ticket to clean this up further (ie, maybe move it elsewhere?).

LGTM Patrick - nice job 👍

Reply:
Yes, both.